index.html

```diff
@@ -16,6 +16,11 @@
             <img src="resources/images/Logo.png" alt="my logo">
         </a>
         <nav class="menu">
+          <ul class="header-menu">
+            <li><a href="index.html"></a>Portfolio</li>
+            <li><a href="index.html"></a>About</li>
+            <li><a href="index.html"></a>Contact</li>
+          </ul>
```

Comment:
- [ ] Please 🙏🏻 make it `display: none;` for your mobile design because it is visible on mobile, kindly adhere to the project requirement because as I can see your project design is `inconsistent` and looks unprofessional, have a look at the below-attached image and table. It is important to make your design consistent no matter for which screen size you're completing your design.

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/Afsheen96/Portfolio/assets/66476329/82f7a29d-aa93-450c-8028-f405e70444b7)

</td>
<td>

![Screenshot from 2023-04-27 19-32-53](https://github.com/Afsheen96/Portfolio/assets/66476329/9ae567ff-7515-4271-83f8-29f652eba09a)


</td>
</tr>
<tr>
<td>

![image](https://github.com/Afsheen96/Portfolio/assets/66476329/74c3c23a-ace5-4257-8011-f4777dcb41f2)


</td>

<td>

![Screenshot from 2023-06-10 13-27-20](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/66476329/0b9964d1-36d3-46d0-b679-abce7a069bb9)


</td>
</tr>
</table>

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)